cnxepub/adapters.py

```diff
@@ -193,6 +194,15 @@ def _make_package(binder):
                 resource = resources.get(filename)
                 if resource:
                     reference.bind(resource, '../resources/{}')
+                else:
+                    if reference.uri.startswith('/contents'):
```

Comment:
Could the `else` case raise an error rather than silently failing?

Aside: Could this test for a uuid regexp? I would really like to get `/contents/` out of the requirements for the HTML and it looks like the resulting link does not contain `/contents/` either